sdk/servicebus/azure-servicebus/samples/sync_samples/send_and_receive_amqp_annotated_message.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
```

Comment:
should this and the async sample be added to the sample readme? Or is it too advanced, so it shouldn't be?

Reply:
added :)